ARTnews.js

```diff
@@ -0,0 +1,147 @@
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2017 czar
+	http://en.wikipedia.org/wiki/User_talk:Czar
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+ 
+function detectWeb(doc, url) { return FW.detectWeb(doc, url); }
+function doWeb(doc, url) { return FW.doWeb(doc, url); }
+ 
+FW.Scraper({
+itemType         : 'blogPost',
+detect           : FW.Url().match(/\/\d{4}\/\d{2}\//),
+title            : FW.Xpath('//h1[@class="entry-title"]').text(),
+attachments      : [{ url: FW.Url(),
+  title:  "ARTnews snapshot",
+  type: "text/html" }],
+creators         : FW.Xpath('//span[@class="author-meta-name"]/a').text().cleanAuthor("author"),
+date             : FW.Xpath('//span[@class="date-meta"]').text(),
+blogTitle        : "ARTnews",
+language         : "en-US",
+abstractNote     : FW.Xpath('//meta[@name="description"]/@content').text(),
+tags             : FW.Xpath('//span[@class="cat-links"]/a').text()
+});
+
+FW.MultiScraper({
+itemType         : 'multiple',
+detect           : FW.Url().match(/(\/category\/)|(\/\?s=)/), // category or search
+choices          : {
+  titles  :  FW.Xpath('//h2[@class="entry-title"]/a').text(),
+  urls    :  FW.Xpath('//h2[@class="entry-title"]/a').key("href").trim()
+  }
+});
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://www.artnews.com/category/2017-venice-biennale/",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "http://www.artnews.com/2016/01/23/venice-biennale-taps-christine-macel-to-be-artistic-director-of-2017-edition/",
+		"items": [
+			{
+				"itemType": "blogPost",
+				"title": "Venice Biennale Taps Christine Macel to Be Artistic Director of 2017 Edition",
+				"creators": [
+					{
+						"firstName": "Andrew",
+						"lastName": "Russeth",
+						"creatorType": "author"
+					}
+				],
+				"date": "01/23/16 10:35 am",
```

Comment:
It'd be good to transform these dates to YYYY-MM-DD to avoid confusion with different locales. I think something like
`.replace(/(\d{2})\/(\d{2})\/(\d{2})/, "20$3-$2-$2")` should work, but please test.